invokeai/backend/config/model_install_backend.py

```diff
@@ -149,10 +149,8 @@ def yes_or_no(prompt: str, default_yes=True):
 def get_root(root: str = None) -> str:
     if root:
         return root
-    elif os.environ.get("INVOKEAI_ROOT"):
-        return os.environ.get("INVOKEAI_ROOT")
     else:
-        return config.root
+        return config.root_dir
```

Comment:
Think my comment here didn't survive - `config.root_dir` is a `Path` object, but this function indicates its return type is always `str`. Let's update the return type, or cast the `Path` to string when returning, whichever is appropriate.

Reply:
Even better. The whole `get_root()` function is never used, so I got rid of it.